src/login/styles/_inputs.scss

```diff
@@ -11,6 +11,56 @@
   }
 }
 
+input[type="radio"] {
+  /* remove standard background appearance */
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  /* create custom radiobutton appearance */
+  display: inline-block;
+  width: 20px;
```

Comment:
are we able to add in the sizes (plus height/widths) as rem, based on 16px?

Reply:
I changed 20px to 1.25rem